src/proxies/panel_manager.js

```diff
@@ -22,9 +22,9 @@ PanelManager.setPoi = async function(poi, options = {}) {
   window.app.unminify();
 };
 
-PanelManager.loadPoiById = async function(id, options) {
-  if (id) {
-    const poi = await ApiPoi.poiApiLoad(id);
+PanelManager.loadPoiById = async function(obj, options) {
```

Comment:
Something gets wrong or unclear with the function name now that you pass full objects…